src/extension/background-script/actions/allowances/__tests__/delete.test.ts

```diff
@@ -9,6 +9,7 @@ const mockAllowances: DbAllowance[] = allowanceFixture;
 const mockPermissions = [
   {
     id: 1,
+    accountId: "123456",
```

Comment:
Shouldn't that be an integer?

Reply:
`accountId`'s are strings
ex: `c2fe55fa-fe99-45da-94d5-ba0271dbabca`

Reply:
Those are created here with `uuidv4`:
https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/actions/accounts/add.ts#L19